frontend/src/components/teacherTaskViewer/index.js

```diff
@@ -1,19 +1,68 @@
 import React from "react";
 import {Link} from "react-router-dom";
-import {Box, Button, Card, Grid, Paper, Table, TableBody, TableCell, TableRow, Typography} from "@material-ui/core";
+import {Box, Button, Card, Grid, Paper, Table, TableHead, TableBody, TableCell, TableRow,  Typography, TextField, ExpansionPanel, ExpansionPanelSummary, ExpansionPanelDetails, Checkbox} from "@material-ui/core";
 import ExpectedOutputBox from "./expectedOutputBox";
 import QuantumSimulator from "../quantum";
 
+import SaveIcon from '@material-ui/icons/Save';
+import DeleteIcon from '@material-ui/icons/Delete';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+
+import { createMuiTheme, ThemeProvider } from '@material-ui/core/styles';
+
+import DoneIcon from '@material-ui/icons/Done';
+import RemoveCircleOutlineIcon from '@material-ui/icons/RemoveCircleOutline';
+import IconButton from '@material-ui/core/IconButton';
+
+const theme = createMuiTheme({
```

Comment:
You shouldn't need to make a new theme, if you want to style something, I recommend you use [makeStyles](https://material-ui.com/styles/basics/#hook-api) instead, it is easier, more powerful and is actually designed for this kind of job. Then again, you can't use it in react component classes and in some cases it might take too much effort to convert the class component into functional components. In this scenario, you may want to look into [our custom styling solution](https://github.com/AndrewDucDanhDo/Software-Engineering-Studio-2A/blob/master/frontend/src/helpers/fashion/index.js).

**But**, it's not too big of a priority to clean this up right now, we can always do it later.